test/cypress/e2e/data/account-test-data.ts

```diff
@@ -0,0 +1,17 @@
+// from mainnet
+const accountHash: string =
```

Comment:
TS is able to infer types, so we don't need to explicitly type here (or line 8 below)

Reply:
Please update all instances in this PR as I notice there are a few others.

Reply:
Updated!